src/sage/matroids/chow_ring.py

```diff
@@ -87,53 +108,61 @@ class ChowRing(QuotientRing_generic):
     EXAMPLES::
 
         sage: M1 = matroids.catalog.P8pp()
-        sage: ch = M1.chow_ring(QQ, False)
+        sage: ch = M1.chow_ring(QQ, False, 'fy')
         sage: ch
-        Chow ring of P8'': Matroid of rank 4 on 8 elements with 8 nonspanning circuits
-        over Rational Field
+        Chow ring of P8'': Matroid of rank 4 on 8 elements with 8 nonspanning
+        circuits in Feitchner-Yuzvinsky presentation over Rational Field
     """
     def __init__(self, R, M, augmented, presentation=None):
         r"""
         Initialize ``self``.
 
         EXAMPLES::
 
-            sage: ch = matroids.Wheel(3).chow_ring(QQ, False)
+            sage: ch = matroids.Wheel(3).chow_ring(QQ, False, 'fy')
             sage: TestSuite(ch).run()
         """
         self._matroid = M
         self._augmented = augmented
         self._presentation = presentation
```

Comment:
```suggestion
        if presentation is None:
            presentation = “fy”
        self._presentation = presentation
```
I think it is useful to have the default being the FY presentation. This is also important for backwards compatibility.